### ISSUE TITLE
Many warnings about missing combiners

### ISSUE THREAD

**Issue by lbergelson** [broadinstitute/gatk#2689]
We get a lot of warnings like this when we run GenomicsDBImport:

Not emitting these is the correct behavior, but it would be nice if we didn't get warnings about it.

@kgururaj @kdatta Could we add a `DO_NOT_EMIT` combiner?  Or really, it would be best to not store these fields at all.

```
WARNING: No valid combination operation found for INFO field DS - the field will NOT be part of INFO fields in the generated VCF records
WARNING: No valid combination operation found for INFO field HaplotypeScore - the field will NOT be part of INFO fields in the generated VCF records
WARNING: No valid combination operation found for INFO field InbreedingCoeff - the field will NOT be part of INFO fields in the generated VCF records
WARNING: No valid combination operation found for INFO field MLEAC - the field will NOT be part of INFO fields in the generated VCF records
WARNING: No valid combination operation found for INFO field MLEAF - the field will NOT be part of INFO fields in the generated VCF records
WARNING: No valid combination operation found for INFO field DS - the field will NOT be part of INFO fields in the generated VCF records
WARNING: No valid combination operation found for INFO field HaplotypeScore - the field will NOT be part of INFO fields in the generated VCF records
WARNING: No valid combination operation found for INFO field InbreedingCoeff - the field will NOT be part of INFO fields in the generated VCF records
WARNING: No valid combination operation found for INFO field MLEAC - the field will NOT be part of INFO fields in the generated VCF records
WARNING: No valid combination operation found for INFO field MLEAF - the field will NOT be part of INFO fields in the generated VCF records
```

**Comment by kgururaj** [comment]
I think we will add a logger for GenomicsDB with configurable verbosity - but this is low priority for us.

**Comment by lbergelson** [comment]
A logger with configurable verbosity would be great, but low priority is fine.  This is a very low priority issue.

**Comment by coreywischmeyer** [comment]
I've just run into this issue, is it okay to ignore these warnings? I'm new to GATK so it's not clear to me what a combination operation is, and if it has effect on the output.

**Comment by kgururaj** [comment]
Yes, as long as you are using the tool to combine gVCF files for the purpose described here: https://gatkforums.broadinstitute.org/gatk/discussion/10061/using-genomicsdbimport-to-consolidate-gvcfs-for-input-to-genotypegvcfs-in-gatk4

**Comment by coreywischmeyer** [comment]
Thanks for answering my question!

**Comment by ldgauthier** [comment]
Part of this arises from the fact that DS and HaplotypeScore were miscategorized when the StandardAnnotation group was made.  I don't believe these have ever been output by HaplotypeCaller and they're not even included in GATK4.

The other part is that MLEAC, MLEAF, and InbreedingCoeff never get combined by any tool -- they get recalculated by GenotypeGVCFs.

I like the suggestion by @lbergelson to make a noop combine operation for the latter.  For the former the real solution is to clean up the header.

**Comment by sooheelee** [comment]
Any update? We have more users asking about the meaning of the WARN.
https://gatkforums.broadinstitute.org/gatk/discussion/13306/genotypegvcfs-warning-message/p1?new=1

**Comment by ldgauthier** [comment]
This is actually becoming quite a problem in the new exome pipeline since there are so many exome intervals and these warnings get output once per query/interval.  The stderr is totally clogged with them and it's impossible to track the task with the GATK ProgressMeter.  @nalinigans I heard you're working on a GDB release -- can we add a quick and dirty logging patch?  One solution would be to make a list of annotations that are recalculated in genotyping that don't need to be merged: InbreedingCoeff, AS_InbreedingCoeff, QD, AS_QD, DS, AC, AF, AN, MLEAC, MLEAF

**Comment by lbergelson** [comment]
@nalinigans
